pkg/configuration/config.go

```diff
@@ -37,6 +37,12 @@ var UserConfigDir string
 // UserConfigFile (e.g. /home/user/doppler/.doppler.yaml)
 var UserConfigFile string
 
+// UserFallbackDir (e.g. /home/user/doppler/.doppler.yaml)
+var UserFallbackDir string
```

Comment:
Would these be better named as `FallbackDir` and `MetadataDir` since they're still used even if the user hasn't configured them? Or am I misunderstanding that part?

Reply:
Good question -- `SetConfigDir` is called in the `init` function with the default `$HOME/.doppler` directory, which is invoked automatically when the module loads. `SetConfigDir` already updates `UserConfigDir` and `UserConfigFile` so I'm mostly just following the existing naming pattern

https://github.com/DopplerHQ/cli/blob/f383368be6389f1f4236abae9d1e12c00e8f2818/pkg/configuration/config.go#L51-L58